rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -932,6 +934,11 @@ stream {
             {{ end }}
 
             {{ if not (empty $location.Backend) }}
+            {{ if $all.DynamicConfigurationEnabled}}
+            log_by_lua_block {
```

Comment:
It seems a bit sketchy that this is in a `location` block. If we use `upstream_balancer` outside of this location we won't call `after_balance`.

Reply:
@csfrancis `proxy_pass http(s)://upstream_balancer` is printed right below at https://github.com/Shopify/ingress/pull/19/files#diff-980db9e4b88704f12338bd074839f94eR942. There's no other place that uses `upstream_balancer`